.travis.yml

```diff
@@ -3,7 +3,8 @@ cache: bundler
 
 bundler_args: --without local_development
 before_install:
-  - gem install bundler
```

Comment:
I don't think we have to install the bundler explicitly, https://github.com/mikel/mail/pull/1308

Reply:
Happy to remove, was just trying to get the tests to pass.